src/app/submission/components/validation-summary/validation-summary.component.html

```diff
@@ -10,16 +10,10 @@
         [summaries]="[
           {
             count: summary.invalidBiomaterials,
-            textOne: 'metadata error',
-            textMany: 'metadata errors',
+            textOne: 'error',
+            textMany: 'errors',
```

Comment:
Is it relevant to use the constant you created throughout this files references to `'error'` and `'errors'`?

Reply:
You mean use an enum for "error" and "errors"

Reply:
I think my pattern matching was a bit off, I thought `'error'` and `'errors'` were submission states.
If they aren't then this is fine as it is.